src/pages/Detail/Detail.jsx

```diff
@@ -1,7 +1,46 @@
 import React from 'react';
+import DetailAside from './components/DetailAside/DetailAside';
+import DetailHeader from './components/DetailArticle/DetailHeader';
+import DetailArticle from './components/DetailArticle/DetailArticle';
+import DetailFooter from './components/DetailArticle/DetailFooter';
+import DetailComments from './components/DetailComments/DetailComments';
 
-function Detail(props) {
-  return <div>여긴 디테일 페이지다.</div>;
+import styled from 'styled-components';
+
+const DetailWrapper = styled.div`
+  display: flex;
+  width: 1140px;
+  margin: 0 auto;
+  margin-top: 80px;
+`;
+
+const Contents = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 60%;
+  padding: 0px 40px;
+`;
+
+const Aside = styled.div`
+  display: flex;
+  width: 40%;
+  border-left: 1px solid ${({ theme }) => theme.border};
+`;
+
+function Detail() {
+  return (
+    <DetailWrapper>
```

Comment:
```suggestion
    <Container>
```
저는 그냥 가장 바깥쪽은 Container로 통일하는 편입니다

Reply:
네 ~ Container로 수정하였습니다!

Reply:
@wjdgotn77 수정하신 내용 push 되었는지 확인해주세요